src/java/org/apache/ivy/osgi/repo/AbstractFSManifestIterable.java

```diff
@@ -44,9 +44,9 @@ public AbstractFSManifestIterable(T root) {
         return new FSManifestIterator();
     }
 
-    protected abstract List<T> listBundleFiles(T dir) throws IOException;
+    protected abstract List<T> listBundleFiles(T dir);
```

Comment:
Similar comment as here https://github.com/apache/ant-ivy/pull/52/files#r127395633 applies to this line